CONTRIBUTING.md

```diff
@@ -52,6 +52,40 @@ If you are maintaining a labwc package for another distro feel free to open an i
 Let's try to stick to sircmpwn's [coding style]. If you're not used to it, you
 can use [checkpatch.pl] to run a few simple formatting checks.
 
+Specific nuances that apply to labwc:
+
+- We prefer `sizeof(*foo)` to `sizeof(struct foo)`
```

Comment:
We prefer `sizeof(*bar)` to `sizeof(struct foo)` for `struct foo *bar;`

We could also add a reasoning here. Badly written suggestions:
- shorter (especially with very long struct names)
- automatically adapts to struct renaming / change of the type of `bar`

Reply:
Probably worth mentioning `znew/znew_for` here also if we decide to add those (related to #552).